### PR TITLE
Add exposed buildings layer

### PR DESCRIPTION
**Pull request by lucernae**
For issue https://github.com/kartoza/fbf-project/issues/123

### CODE REVIEW

File: app-dashboard/js/view/map.js [hunk 1]
```diff
@@ -213,9 +225,47 @@ define([
...
+                'sub_district': 'sub_district_id',
+                'village': 'village_id',
+            }
+
+            this.exposed_layers = this.depth_class_collection.map(function (depth_class) {
+                let label = `Exposed Buildings in Depth Class: ${depth_class.get('label')}`
+                let exposed_layer = L.tileLayer.wms(
+                    geoserverUrl,
+                    {
+                        layers: `kartoza:exposed_buildings`,
```
Comment (timlinux) [inline app-dashboard/js/view/map.js:265]: I'm not seeing this layer in GeoServer:
image

File: app-dashboard/js/view/map.js [hunk 2]
```diff
@@ -213,9 +225,47 @@ define([
...
+                        format: 'image/png',
```
Comment (timlinux) [inline app-dashboard/js/view/map.js:266]: I wonder if we can get a performance / size improvement by using GIF?

File: app-dashboard/js/view/map.js (hunk 1 above)
Comment (lucernae) [reply app-dashboard/js/view/map.js:265]: Correct @timlinux , I advised in the issue #123 to let @NyakudyaA help make the layer.
But the spec should be the same with I described in the issue. So we can just enable it.

File: app-dashboard/js/view/map.js (hunk 2 above)
Comment (lucernae) [reply app-dashboard/js/view/map.js:266]: We can easily switch later to try. I've setup most WMS request with easier configuration like this.

**Comment by lucernae** [comment]
I will merge this for now to avoid future conflicts. We can enable the functionality later when the layer is ready

**Pull request merged**